variables.env

```diff
@@ -39,6 +44,7 @@ WDQS_VERSION=0.3.137
 # We cannot change this at the moment
 ELASTICSEARCH_VERSION=7.10.2
 ELASTICSEARCH_IMAGE_URL=docker.elastic.co/elasticsearch/elasticsearch:7.10.2
+# TODO: do we need ELASTICSEARCH_BUNDLE_VERSION so that the tag can express plugin changes?
```

Comment:
I am not sure I got your question, but if I'm getting you-- yes, I think this image should be versioned like: 

1.0.0+es-7.10.2

I don't think this image currently will diverge between our 3 different Wikibase versions we're releasing (it hasn't yet yet that I know of), so it would just be that version and build for all 3 releases.

Reply:
Yes, I think you got it, but just for reference, the idea was to add a variable which versions all elastic search components together, essentially hiding the upstream versions, so that we have more control in expressing the type of changes.